_content/excerpts/of-theras-and-theris_roy-kumkum.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Of Theras and Therīs: Visions of liberation in the Early Buddhist Tradition"
+authors:
+  - "KumKum Roy"
+drive_links:
+  - "https://drive.google.com/file/d/1k79UxZwl7sqXs94iAWzfrJ_1xdRW80OG/view?usp=sharing"
```

Comment:
Here's the external url: `http://14.139.58.200/ojs/index.php/shss/article/view/591/560`  Though it is a slightly earlier version.